src/mds/CDir.cc

```diff
@@ -2234,7 +2234,7 @@ void CDir::_omap_commit(int op_prio)
     for (auto p = items.begin(); p != items.end(); ) {
```

Comment:
let assert fragmenting difrag's committing_version is 0. Otherwise, looks good.

Reply:
The fragmenting difrag's committing_version is bigger than 0, because the basedirfrag's version has been set to the maximum version of the origin dirfrag. Thandks!

// merge version
if (dir->get_version() > get_version())
set_version(dir->get_version());

Reply:
sorry. I mean committed_version

Reply:
fixed it ,thanks!

Reply:
Have move assert out of the loop,thanks.